docs/api-docs/tax/tax-properties.md

```diff
@@ -0,0 +1,375 @@
+# Tax Properties for Products
+
+Tax properties are inputs that merchants can provide into a tax provider's tax calculation to receive more accurate tax calculations from third-party tax providers. These properties are specific to a tax provider and allow merchants and shoppers to have tax-related fields factored into their tax rates. 
+
+The Tax Property V3 API will allow you to perform the following functions:
+- Add tax property fields that are specific to a tax provider (have **variable** inputs for tax calculations)      
+- Associate tax properties to **products**. Unlike tax codes, you can add **multiple** tax properties to a single product
+
+<!-- theme:info -->
+> #### Note
+> Currently, you can associate tax properties to products at the **product level**. 
```

Comment:
I assume we use language such as "products at the variant level" elsewhere? 👍

Reply:
Changed the wording to base products to match KB docs